mixer/adapter/skywalking/protocol/service-mesh.proto

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Asf license? Either include the service def from source or have an istio license